test/index.js

```diff
@@ -37,13 +44,16 @@ describe('Bananas', () => {
         };
 
         let updates = [];
-        const orig = Wreck.post;
         Wreck.post = function (uri, options, next) {
 
+            expect(uri).to.equal('https://logs-01.loggly.com/bulk/abcdefg');
+            expect(options.json).to.be.true();
+            expect(options.headers).to.equal({ 'content-type': 'application/json' });
             updates = updates.concat(options.payload.split('\n'));
             return next();
         };
 
+        const timeBeforeBananasRegister = new Date();
```

Comment:
OK

Do you think it's over-testing?

in the old tests the following code will pass:
```javascript
internals.update = function (event, request) {

    const now = Date.now();

    const update = {
        event: event,
        timestamp: "now",
        host: Os.hostname()
    };
// ...
```